go.mod

```diff
@@ -12,10 +12,10 @@ require (
 	github.com/palantir/pkg/metrics v1.0.1
 	github.com/palantir/pkg/objmatcher v1.0.1
 	github.com/palantir/pkg/safejson v1.0.1
-	github.com/palantir/pkg/safeyaml v1.0.1
 	github.com/palantir/pkg/signals v1.0.1
 	github.com/palantir/pkg/tlsconfig v1.0.1
 	github.com/palantir/witchcraft-go-error v1.4.3
+	github.com/palantir/witchcraft-go-health v0.0.0-20201110010813-1ffc140553de
```

Comment:
We should release a tagged version of WGH and bump to depend on that before merging this.